app/models/mailchimp_synchronizer.rb

```diff
@@ -162,6 +163,23 @@ def get_gender_translation (contact)
   def get_coefficient_translation (contact)
     contact.coefficients.where(account_id: account.id).first.try(:value).try(:to_s) || ''
   end
+
+  # TODO primero lo cableo para probarlo, luego lo hago de forma copada
+  def get_followers_for(contact)
+    response = RestClient.get PADMA_CRM_HOST + '/api/v0/follows/followed_by',
```

Comment:
jej, ok.


*---Dwayne*

On Wed, Mar 30, 2016 at 12:24 PM, afalkear <notifications@github.com> wrote:

> In app/models/mailchimp_synchronizer.rb
> <https://github.com/dwaynemac/contacts/pull/10#discussion_r57907871>:
>
> > @@ -162,6 +163,23 @@ def get_gender_translation (contact)
> >    def get_coefficient_translation (contact)
> >      contact.coefficients.where(account_id: account.id).first.try(:value).try(:to_s) || ''
> >    end
> > +
> > +  # TODO primero lo cableo para probarlo, luego lo hago de forma copada
> > +  def get_followers_for(contact)
> > +    response = RestClient.get PADMA_CRM_HOST + '/api/v0/follows/followed_by',
>
> si, ahora lo cambio a typhoeus. y saco ese comentario de TODO :P
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/dwaynemac/contacts/pull/10/files/0caecf7799997e64bbfeacec61a620dd31645df9#r57907871>
>